traefik/templates/_podtemplate.tpl

```diff
@@ -134,6 +134,9 @@
           {{- if .Values.providers.kubernetesCRD.enabled }}
           - "--providers.kubernetescrd"
           {{- end }}
+          {{- if .Values.providers.kubernetesCRD.allowCrossNamespace }}
```

Comment:
because of the way Traefik flags works setting 
```yaml
providers:
  kubernetesCRD:
    enabled: false
    allowCrossNamespace: true
```
Would actually enable the CRD provider, which is weird